config.toml.sample

```diff
@@ -20,7 +20,7 @@ debug = false                           # Please refer to `[Debug mode](https://
 maskUserInfo = false                    # If true, then account information in header, session list will be masked
 #singleSignOnVendors = "saml"           # Comma-separated list of vendors that support single sign-on.
 enableContainerCommit = false           # If true, then user can request container commit corresponding to the session in running status
-
+hide-agents = true                      # If false, show the `Agent Summary` menu in the sidebar.
```

Comment:
Perhaps there's key mismatching in `backend-ai-webui` component [here](https://github.com/lablup/backend.ai-webui/pull/1439/files#diff-c970ee927d9c8e6050388af9c74159ccfd53cdff999e3e89f836009010c01aabR334-R336).

Reply:
Oops! duplicated comments! just refer to the first one, please. 😅

Reply:
ah I didn't reflect on this file. Thanks! (I didn't recognize it because I changed `config.toml` 😅)